packages/mdc-ripple/util.ts

```diff
@@ -86,30 +80,6 @@ export function supportsCssVariables(windowObj: Window, forceRefresh = false): b
   return supportsCssVars;
 }
 
-/**
- * Determine whether the current browser supports passive event listeners, and
- * if so, use them.
- */
-export function applyPassive(globalObj: Window = window, forceRefresh = false):
```

Comment:
Seems like only IE does not support `{passive: boolean}` option. Doesn't the browser just ignore this option if not supported. Not sure why we've introduced this check in first place. WDYT?

Reply:
I looked into it - https://developers.google.com/web/tools/lighthouse/audits/passive-event-listeners

> However, in browsers that do not support passive event listeners, the third parameter is a boolean to indicate whether the event should bubble or capture. So, using the syntax above may cause unintended consequences.

I think we need to leave this in here if I understand correctly.